### PR TITLE
feat: add migration block

### PR DESCRIPTION
**Pull request by shakirckneoito**
Added a new block for handling database migrations , 
Migrations are handle using umzug package.

example
```
npm run migrate:up
npm run migrate:down
npm run migrate:create
npm run migrate filename

```

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
+import { Sequelize } from "sequelize";
+import AWS from "aws-sdk";
```
Comment (muhammadtsneo) [inline index.js:2]: can we update the import here to
import { SecretsManager } from 'aws-sdk', 

will this affect aws sdk initialisation

**Review by muhammadtsneo — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (shakirckneoito) [reply index.js:2]: I don't think changing that import will affect anything. will update it

Comment (shakirckneoito) [reply index.js:2]: @muhammadtsneo  We cannot import Secretsmanager like this!!

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+/node_modules
```
Comment (mohdmusthafaneo) [inline .gitignore]: ignore .log files too

**Review by mohdmusthafa — APPROVED** [review]
LGTM. Good work! 🥇 .

**Review by mohdmusthafaneo — APPROVED** [review]

**Pull request merged**